frontend/ui/widget/configdialog.lua

```diff
@@ -591,20 +610,32 @@ function ConfigOption:init()
                     num_buttons = #self.options[c].values,
                     position = self.options[c].default_pos,
                     callback = function(arg)
+
                         if arg == "-" or arg == "+" then
+                            Notification:setNotifySource(Notification.SOURCE_BOTTOM_MENU_FINE)
                             self.config:onConfigFineTuneChoose(self.options[c].values, self.options[c].name,
                                 self.options[c].event, self.options[c].args, self.options[c].events, arg, self.options[c].hide_on_apply,
                                 self.options[c].fine_tune_param)
                         elseif arg == "⋮" then
+                            Notification:setNotifySource(Notification.SOURCE_BOTTOM_MENU_MORE)
                             self.config:onConfigMoreChoose(self.options[c].values, self.options[c].name,
                                 self.options[c].event, arg, name_text, self.options[c].more_options_param)
                         else
+                            if self.options[c].buttonprogress then
+                                Notification:setNotifySource(Notification.SOURCE_BOTTOM_MENU_PROGRESS)
+                            else
+                                Notification:setNotifySource(Notification.SOURCE_BOTTOM_MENU_TOGGLE)
+                            end
```

Comment:
Looks like we already in a `if self.options[c].buttonprogress` then branch (from line 597).

Reply:
Indeed